siftseek/logging_config.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+
+def configure_logging() -> None:
+    """
+    Configures application-wide logging with a rotating file handler.
+
+    This function sets up logging to write messages to a file, ensuring that log
+    files are rotated when they reach a certain size. It checks if a 'logs' directory
```

Comment:
this seems like kind of unnecessary added complexity. Most companies have all of their logs written to std_out and have a log aggregating tool to collect this information. 10kb is also very small.

Reply:
Ok, good to know!